app/controllers/identities_controller.rb

```diff
@@ -69,4 +80,29 @@ def show_sites # rubocop:disable Metrics/AbcSize, Metrics/CyclomaticComplexity,
       render plain: "404 Not Found", status: :not_found
     end
   end
+
+  private
+
+  def flipper_params
+    params.require(:portal).to_sym
+  end
+
+  def form_user
+    identity = current_user.identity_scope.find_by(uid: params[:id])
+    identity.user.becomes(FormUser)
+  end
+
+  def admin?
```

Comment:
good idea extracting admin? to its own method. Do you think it would be useful to have this method available at a higher level so it could be re-used in other controllers too?

Reply:
great, will move to app controller! was not sure how liberal should I go with touching someone's code 😄

Reply:
done, moved into app ctrlr